docs/metrics/learning-prometheus-server/README.md

```diff
@@ -0,0 +1,184 @@
+
+Let's create a new file in the same location as `prometheus.yml`, called
+`otel.yml` for this exercise. Copy and paste the entire content below into the
+otel.yml file we have created just now.
+
+```yaml
+global:
+  scrape_interval: 10s
+  scrape_timeout: 10s
+  evaluation_interval: 10s
+scrape_configs:
+- job_name: MyOpenTelemetryDemo
+  honor_timestamps: true
+  scrape_interval: 1s
+  scrape_timeout: 1s
+  metrics_path: /metrics
+  scheme: http
+  follow_redirects: true
+  static_configs:
+  - targets:
+    - localhost:9184 # set the port that prometheus server will listen to
+```
+
+### Start Prometheus
+
+Follow the instructions from
+[starting-prometheus](https://prometheus.io/docs/introduction/first_steps/#starting-prometheus)
+to start the Prometheus server and verify it has been started successfully.
+
+Please note that we will need pass in otel.yml file as the argument:
+
+```console
+./prometheus --config.file=otel.yml
+```
+
+Once the server is started, we are going to make some small tweaks to the
+example in the getting-started metrics [example](../getting-started/Program.cs)
+to export our metrics to the endpoint that prometheus was configured to listen
+to.
+
+### Configure OpenTelemetry to Expose metrics to Prometheus Endpoint
+
+Create a new console application and run it:
+
+```sh
+dotnet new console --output prometheus-http-server
+cd prometheus-http-server
+dotnet run
+```
+
+We will have to add a reference to prometheus exporter to the
+prometheus-http-server application.
+
+```shell
+dotnet add package OpenTelemetry.Exporter.Prometheus --version 1.2.0-rc1
+```
+
+Now, replace the below line in the [Program.cs](../getting-started/Program.cs)
+
+```csharp
+.AddConsoleExporter()
+```
+
+with
+
+```csharp
+.AddPrometheusExporter(opt =>
+{
+    opt.StartHttpListener = true;
+    opt.HttpListenerPrefixes = new string[] { $"http://localhost:9184/" };
+})
+```
+
+We set the options for our prometheus exporter to export data via the endpoint
+that we've configured Prometheus server to listen to in the otel.yml file.
+
+Also, for our learning purpose, use a while-loop to keep increasing the counter
+value until any key is pressed.
+
+```csharp
+Console.WriteLine("Press any key to exit");
+while (!Console.KeyAvailable)
+{
+    Thread.Sleep(1000);
+    MyFruitCounter.Add(1, new("name", "apple"), new("color", "red"));
+    MyFruitCounter.Add(2, new("name", "lemon"), new("color", "yellow"));
+    MyFruitCounter.Add(1, new("name", "lemon"), new("color", "yellow"));
+    ...
+    ...
+    ...
+}
+```
+
+After the above modifications, now our `Program.cs` should look like [this](./Program.cs).
+
+### Check Results in Prometheus
+
+Start the application and leave the process running. Now we
+should be able to see the metrics at the endpoint we've configured in
+`otel.yml` and defined in `Program.cs`; in this case, the endpoint
+is: "http://localhost:9184/".
+
+Check the output metrics with your favorite browser:
```

Comment:
the title says check results in Prometheus, so this can be bit confusing.

Reply:
another alternate way of arranging the tutorial:
1. Start with console.
2. Add Prometheus exporter.
3. Use browser to see the metrics.
----until this , no Prometheus installation required----
4. Install Prometheus, and ask it to scrape the metrics
5. Use prometheus to show metrics
6. Make things more colorful by using Grafana :)

Reply:
I've restructured the file a bit, please check it out to see if the flow makes more senses now. Thank you!